### ISSUE TITLE
Issue while using 8-bit architecture modules

### ISSUE THREAD

**Issue by Krishanu-007** [flandreas/antares#806]
Continuous popping of System Malfunction, whenever any 8-bit module is being used forcing the app to restart

**Comment by Krishanu-007** [comment]
[Antares.log](https://github.com/user-attachments/files/16728271/Antares.log)
[currentStorable.txt](https://github.com/user-attachments/files/16728272/currentStorable.txt)

**Comment by flandreas** [comment]
@Krishanu-007 Thanks for reporting.

You seem to have exported a system dump file. It should be named something like Antares-dump.zip. Could you please attach it to this issue? I'll need it for analysis.

**Comment by Krishanu-007** [comment]
[Antares-dump.zip](https://github.com/user-attachments/files/16729683/Antares-dump.zip)

**Comment by flandreas** [comment]
Thanks! I can reproduce the issue. I still have to find its cause, and then I'll release a fix as soon as possible..

**Comment by flandreas** [comment]
@Krishanu-007 Seems to be a "false positive" of the consistency checker infrastructure. This has to be fixed.

In the meantime, I've disabled the consistence checker. Disabling is controlled by a file Antares reads from the GitHub repository. Make sure to restart Antares and try again. I've just tried it with your project (contained in the dump you've posted), and I was able to successfully add the "Multiplier n bits" component to your circuit "8-bit Arithmetics".
